### DISCUSSION TITLE
Change counter intuitiv horizontal block filling in the dashboard to vertical one

### DISCUSSION THREAD

**Discussion by RequestPrivacy** [mempool/mempool#562] [Ideas]
### Description

For me, a new block (and in general, the mempool) is always "filled" with transactions. 

The current graphical depiction of this process on the dashboard is an increasing horizontal filling from right to left with a color gradient from orange to green. I get the point of showing the future growth of the blockchain horizontally, but it feels kind of weird to me - especially if not fully loaded, but mined blocks are displayed with the more intuitive "vertical" filling with an empty space on top (and in doing so guaranteeing a continuous colored horizontal link to the neighboring blocks). 

### Question / Suggestion

Do other people find this minor (for me) issue also counter intuitive, or am I the only one? For me a continuous vertical filling of all blocks would make more sense.

### Screenshots

![Block_filling](https://user-images.githubusercontent.com/72392544/120523774-d7b6ff00-c3c5-11eb-9ab2-1396d11a63f2.png)

**Comment by softsimon** [comment]
In the beginning all blocks filled up vertically.

But after adding the color of the mempool filling up, and with transaction tracking arrow that shows you exactly where in the block your transaction is located, it has to show the filling up horizontally.

So to make it consistent it must be the mined blocks that should flip to horizontal... I didn't do it because it doesn't look as good and the mempool logo itself is a bucket filling up vertically.

<img width="1293" alt="Screen Shot 2021-06-02 at 13 26 43" src="https://user-images.githubusercontent.com/8561090/120525538-34a2bd00-c3a6-11eb-8902-28c0f6a077cf.png">

**Reply by RequestPrivacy** [reply]
Sorry for my late reply and thanks for the clear explanation. It helped me to see the trade-offs and why you made the choice!
